CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ if (APPLE)
 endif()
 
 if(WIN32)
+	set(MINGW_SYSTEM_PREFIX "C:/msys64/mingw64" CACHE PATH "Path to the mingw-w64 system root.")
```

Comment:
mingw isn't just available for windows please set the paths aswell for unix/osx

Reply:
That'd be a tad different because we'd be cross-compiling, though? Also, it's just the default. In fact, if you look at the CI run, I'm explicitly overriding this path.

Reply:
You're correct, then we'd be cross-compiling (which still should work i guess)
Ahh you have to define it to override it?

Reply:
> Ahh you have to define it to override it?

I don't think so, actually. But the idea was precisely to make it easier to use on a vanilla msys2 install (so by default it uses the default installation path on windows). You just made me realize a bug though because it's missing the `CPACK_` prefix.